docs/pages/x/api/date-pickers/pickers-layout.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import ApiPage from 'docsx/src/modules/components/ApiPage';
+import mapApiPageTranslations from 'docs/src/modules/utils/mapApiPageTranslations';
+import jsonPageContent from './pickers-layout.json';
+
+export default function Page(props) {
+  const { descriptions, pageContent } = props;
+  return <ApiPage descriptions={descriptions} pageContent={pageContent} />;
+}
+
+Page.getInitialProps = () => {
+  const req = require.context(
+    'docsx/translations/api-docs/date-pickers',
+    false,
+    /\/pickers-layout(-[a-z]{2})?\.json$/,
```

Comment:
Same for this one? Also, it does not seem to be linked anywhere (has no demos), maybe it's worth adding it in `custom-layout`?